index.html

```diff
@@ -1,32 +1,164 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio Setup</title>
     <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    <section>
-        <header>
-            <div class="logo"><a href="#"></a>Ali-Tech</div>
-            <nav class="navbar"><img src="/img/Union.png" alt=""><a href="#"></a></nav>
-        </header>
-        <section class="main">
-            <h1 class="heading">I’m Ali Raza.<br>
-                Glad to see <br>you!</h1>
-            <p class="sub">I’m a software developer! I can help you build a product , feature or website Look through some
-                of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
-                contact me.</p>
-                <p class="contact"><a href="#">LET’S CONNECT</a></p>
-                <ul class="icon">
-                    <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt=""></a></li>
-                    <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt=""></a></li>
-                    <li class="m"><a href="#"><img src="/img/m.svg" alt=""></a></li>
-                    <li class="github"><a href="#"><img src="/img/github.svg" alt=""></a></li>
-                    <li class="v"><a href="#"><img src="/img/v.svg" alt=""></a></li>
-                </ul>
-        </section>
-    </section 
+    <header>
+        <div class="logo"><a href="#"></a>Ali-Tech</div>
+        <nav class="navbar"><img src="/img/Union.png" alt=""><a href="#"></a></nav>
+    </header>
+    <section class="main">
+        <h1 class="heading">I’m Ali Raza.<br>
+            Glad to see <br>you!</h1>
+        <p class="sub">I’m a software developer! I can help you build a product , feature or website Look through
+            some
+            of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+            contact me.</p>
+        <ul> 
+         <li><a href="#"><img src="/img/LET’S CONNECT.svg" alt="" class="contact"></a></li>
+        </ul>
+        <ul class="icon">
+            <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt=""></a></li>
+            <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt=""></a></li>
+            <li class="m"><a href="#"><img src="/img/m.svg" alt=""></a></li>
+            <li class="github"><a href="#"><img src="/img/github.svg" alt=""></a></li>
+            <li class="v"><a href="#"><img src="/img/v.svg" alt=""></a></li>
+        </ul>
+    </section>
+    <section class="workcards">
+        <div class="card card1">
+         <div class="pic1"><img src="/img/Snapshoot Portfolio (1).png" alt="Snapshoot"></div>
+              <img src="/img/Project Title (1).svg" alt="tonic" class="title">
```

Comment:
- [ ] Please make sure to use a header tag, `hx`, for project. Using header tag for the project title ensures accessibility and SEO ✨⛏ Because of using an image, the project's title is looking different from the one given in the Figma template

![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/ef42a24f-afdb-47ec-a0cb-e8c9615c02ed)


_Kindly apply this to all project cards_

Reply:
@KanzaTahreem Hi I am trying to add an h2 tag and give the same values given in the template but the result are not same
![Screenshot_1](https://github.com/ali-techworld/Portfolio-Setup/assets/119800035/277a1af6-eb76-4704-afe8-96f270b010e3)

Reply:
@KanzaTahreem that's why I use the pic can you help me

Reply:
Hello @ali-techworld, that's because the `font-weight` is not imported. Try it by replacing your line 4 of `style.css` with the following
`@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700;800&family=Poppins:wght@400;500;600;700;800;900&display=swap');
` This is what I'm getting with the correct styles 👇
  
  ![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/95db9c87-0aaa-4887-acea-99cdc1283cdf)